src/devtools/App.vue

```diff
@@ -60,7 +60,8 @@ export default {
   computed: mapState({
     message: state => state.message,
     tab: state => state.tab,
-    newEventCount: state => state.events.newEventCount
+    newEventCount: state => state.events.newEventCount,
+    isDark: () => chrome.devtools && chrome.devtools.panels.themeName === 'dark'
```

Comment:
A few small things:

- The devtools code itself doesn't assume it's always running in Chrome. We need to check `typeof chrome !== 'undefined'` first.

- `isDark` doesn't really need to be a computed property. It's initialized only once and shouldn't be in `mapState` either. Let's move it to `data()` instead.